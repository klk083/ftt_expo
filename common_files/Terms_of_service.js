import React from 'react'
import { View, ScrollView, Text, StyleSheet, SafeAreaView } from 'react-native'
import Constants from "expo-constants";

import {terms_of_service} from './Texts'
import { RFPercentage } from "react-native-responsive-fontsize";


export default class Privacy extends React.Component {
    render() {
        return (
            <SafeAreaView style={styles.safeAreaView}>
            <View style={styles.container}>
                <ScrollView>
                    <Text style={styles.text}>{terms_of_service}</Text>
                </ScrollView>
            </View>
            </SafeAreaView>
        );
    }
}

const styles = StyleSheet.create({
    safeAreaView: {
        flex: 1,
    },
    container: {
        alignItems: 'center',
        justifyContent: 'center',
        marginLeft: 10,
        marginRight: 10,
        marginTop: 10,
    },
    text: {
        marginTop: 20,
        fontSize: RFPercentage(3)
    }
});

