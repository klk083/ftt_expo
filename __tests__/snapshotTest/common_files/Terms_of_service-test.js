import React from 'react'
import Terms_of_service from '../../../common_files/Terms_of_service'

import renderer from 'react-test-renderer'

test('renders correctly', () => {
    const tree = renderer.create(<Terms_of_service />).toJSON()
    expect(tree).toMatchSnapshot()
})
