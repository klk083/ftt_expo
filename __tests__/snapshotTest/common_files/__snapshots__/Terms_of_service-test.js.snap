// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "justifyContent": "center",
        "marginLeft": 10,
        "marginRight": 10,
        "marginTop": 10,
      }
    }
  >
    <RCTScrollView>
      <View>
        <Text
          style={
            Object {
              "fontSize": 40,
              "marginTop": 20,
            }
          }
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec in tortor libero. Nullam quis ante ut velit gravida porttitor. Curabitur gravida accumsan nisl. Integer sit amet dui gravida, scelerisque massa eu, venenatis sem. Sed posuere augue non diam ornare venenatis non vel tellus. Suspendisse a metus imperdiet, ornare nibh non, gravida dui. Nulla facilisi. Vivamus convallis ac felis at dignissim. Integer ultricies aliquet viverra. Suspendisse quis placerat justo. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Aliquam a vehicula nunc, sed sagittis mi. Suspendisse potenti. Phasellus facilisis nunc mauris, ut porttitor est bibendum eget.

Sed luctus vel lacus ut convallis. Nunc laoreet accumsan orci, quis mattis augue. Nulla ipsum nibh, egestas a sagittis non, tempus vitae massa. Pellentesque ullamcorper, sem malesuada accumsan porta, ex lectus dictum nisi, a bibendum justo sapien id sapien. Etiam ut fermentum libero. Pellentesque id urna sem. Duis sit amet hendrerit magna, eu placerat risus. Praesent vel egestas diam. Maecenas vulputate ante leo, ut viverra nisi laoreet a. Curabitur rhoncus in tellus sit amet porttitor. Curabitur quis ex ornare, tincidunt nulla et, luctus augue. Duis malesuada, metus et ornare sollicitudin, erat augue fringilla arcu, non fringilla lacus nisi vitae ipsum. Nulla eu tellus quis est sollicitudin efficitur.

Morbi feugiat congue dolor. Duis auctor tristique erat, in cursus ligula condimentum quis. Praesent gravida neque in malesuada suscipit. Sed sit amet risus sed lectus lobortis venenatis a ut sapien. Suspendisse potenti. Donec posuere massa elit, ac posuere dui sodales vel. Vestibulum luctus sit amet erat ac eleifend. Suspendisse potenti. Morbi facilisis ex ligula, eget aliquam ex elementum eget. Quisque ac blandit massa. Fusce id orci vel ante porta ultrices. Nullam eget leo eget nunc mollis mollis. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Suspendisse non elementum dui.

Nullam et urna id orci laoreet porttitor a in libero. Nullam sollicitudin ultrices pellentesque. Maecenas eu ullamcorper justo. Fusce at vulputate enim. Cras pellentesque sed sapien at cursus. In rhoncus mollis libero, ac finibus metus porttitor eu. Nam vestibulum, lacus sit amet varius malesuada, nisi ipsum semper quam, non rutrum sem nisl ut sem. Nam ut elit venenatis, malesuada risus sollicitudin, consequat dolor. Ut vitae purus diam. Proin condimentum nisi eu lacus pharetra ultrices. In hac habitasse platea dictumst. Mauris malesuada nisl in nunc varius imperdiet. Praesent id augue viverra, viverra lectus non, vehicula tortor.

Donec bibendum sapien sed sem condimentum, ac interdum leo molestie. Duis accumsan aliquet finibus. Fusce luctus tempus bibendum. Proin iaculis egestas turpis, vitae sodales lectus pharetra at. Nam posuere arcu diam, eget ultricies ligula imperdiet congue. Etiam volutpat volutpat elit eget molestie. In mattis eu ligula eu sodales. Mauris eget orci varius, pharetra quam a, eleifend justo. Integer dapibus erat ac orci eleifend, eget ornare orci ullamcorper. Curabitur diam urna, dignissim ut risus eget, fringilla fermentum augue. Praesent faucibus, mi at accumsan iaculis, purus ante molestie nibh, at egestas velit arcu id felis. Etiam eget cursus massa, in sagittis erat. Suspendisse condimentum ex velit, ut pharetra magna faucibus in. Maecenas congue lacus massa, ac tincidunt ZZerat viverra condimentum.
        </Text>
      </View>
    </RCTScrollView>
  </View>
</RCTSafeAreaView>
`;
